vector/src/main/res/values/strings.xml

```diff
@@ -1708,20 +1708,20 @@
     <string name="settings_discovery_identity_server_info">You are currently using %1$s to discover and be discoverable by existing contacts you know.</string>
     <string name="settings_discovery_identity_server_info_none">You are not currently using an identity server. To discover and be discoverable by existing contacts you know, configure one below.</string>
     <string name="settings_discovery_emails_title">Discoverable email addresses</string>
-    <string name="settings_discovery_no_mails">Discovery options will appear once you have added an email.</string>
+    <string name="settings_discovery_no_mails">Discovery options will appear once you have added an email address.</string>
     <string name="settings_discovery_no_msisdn">Discovery options will appear once you have added a phone number.</string>
     <string name="settings_discovery_disconnect_identity_server_info">Disconnecting from your identity server will mean you won’t be discoverable by other users and you won’t be able to invite others by email or phone.</string>
     <string name="settings_discovery_msisdn_title">Discoverable phone numbers</string>
-    <string name="settings_discovery_confirm_mail">We sent you a confirm email to %s, check your email and click on the confirmation link</string>
-    <string name="settings_discovery_confirm_mail_not_clicked">We sent you a confirm email to %s, please first check your email and click on the confirmation link</string>
+    <string name="settings_discovery_confirm_mail">We sent an email to %s. Check your email and click on the confirmation link.</string>
+    <string name="settings_discovery_confirm_mail_not_clicked">We sent an email to %s. Check your email and click on the confirmation link.</string>
```

Comment:
`settings_discovery_confirm_mail_not_clicked` value has to be different than `settings_discovery_confirm_mail`.

`settings_discovery_confirm_mail_not_clicked` is used when the user wants to continue the process to make their email discoverable, but did not click on the link in the email. 

Currently we have:

1. First displayed screen

<img width="283" alt="Screenshot 2022-03-10 at 13 16 35" src="https://user-images.githubusercontent.com/3940906/157660415-13407647-81a2-4bd7-9109-add673812b66.png">

2. User clicks on "Continue" (but did not click on the link in the email)

<img width="284" alt="Screenshot 2022-03-10 at 13 18 16" src="https://user-images.githubusercontent.com/3940906/157660430-499e26dc-7d14-4430-b6eb-f19917649a4e.png">